### PR TITLE
YSQL Datamodeling

### PR DESCRIPTION
**Pull request by premkumr**
- Added a new page for YSQL Data modeling in Develop
- Removed a similar page in yb-voyager
- Setup redirects to the new page

### CODE REVIEW

File: docs/content/preview/develop/learn/data-modeling-ysql.md [hunk 1]
```diff
@@ -1,17 +1,373 @@
...
+
 {{<api-tabs>}}
 
-Data modeling is a process that involves identifying the entities (items to be stored) and the relationships between entities. To create your data model, identify the patterns used to access data and the types of queries to be performed. These two ideas inform the organization and structure of the data, and the design and creation of the database's tables.
+Data modeling involves designing the database schema for efficient storage and access. In a distributed SQL database like YugabyteDB, table data is split into tablets and distributed across multiple nodes within the cluster, allowing applications to connect to any node for storing and retrieving data. Reads and writes can span multiple nodes, so it's crucial to consider how table data is partitioned and distributed when modeling your data.
+
+In YugabyteDB, data is stored as rows and columns in tables which are organized under schemas and databases.
+
+{{<lead link="../../../explore/ysql-language-features/databases-schemas-tables">}}
+To understand more about creating and managing tables, schemas and databases, see [Schemas and tables](../../../explore/ysql-language-features/databases-schemas-tables)
```
Comment (ddhodge) [inline docs/content/preview/develop/learn/data-modeling-ysql.md:21]:
```suggestion
To understand more about creating and managing tables, schemas, and databases, see [Schemas and tables](../../../explore/ysql-language-features/databases-schemas-tables).
```

File: docs/content/preview/develop/learn/data-modeling-ysql.md [hunk 2]
```diff
@@ -1,17 +1,373 @@
...
+{{</lead>}}
+
+When modeling your data for YugabyteDB, you must understand the 2 [data distribution](../../../explore/going-beyond-sql/data-sharding) schemes, Hash and Range sharding in detail. This is very significant for designing your tables and indexes for fast retrieval and storage.
```
Comment (ddhodge) [inline docs/content/preview/develop/learn/data-modeling-ysql.md:24]:
```suggestion
To design your tables and indexes for fast retrieval and storage in YugabyteDB, you first need to understand the two [data distribution](../../../explore/going-beyond-sql/data-sharding) schemes, Hash and Range sharding, in detail.
```

File: docs/content/preview/develop/learn/data-modeling-ysql.md [hunk 3]
```diff
@@ -1,17 +1,373 @@
...
+
+In YugabyteDB the sharding and ordering of data in the tables and indexes is governed by the primary key of the table and index respectively. Let us dig deeper into designing the primary keys with some examples.
```
Comment (ddhodge) [inline docs/content/preview/develop/learn/data-modeling-ysql.md:26]:
```suggestion
In YugabyteDB, the sharding and ordering of data in the tables and indexes is governed by the primary key of the table and index respectively.
```

File: docs/content/preview/develop/learn/data-modeling-ysql.md [hunk 4]
```diff
@@ -1,17 +1,373 @@
...
+  (34,'William',47,94085,False),  (35,'William',60,94085,True),  (36,'Sarah',53,94085,False),
+  (37,'Ashley',47,94084,True),    (38,'Ashley',54,94084,False),  (39,'Benjamin',42,94083,False),
+  (40,'Tyler',47,94085,True),     (41,'Michael',42,94084,False), (42,'Diane',50,94084,False),
+  (43,'Nancy',51,94085,False),    (44,'Rebecca',56,94085,False), (45,'Tyler',41,94085,True);
+```
+
+</details>
+
+{{<note>}}
+To explain the behavior of the queries we will use **explain (analyze, dist, costs off)**. In practice, you do not need this unless you are trying to optimize the performance. For more details, see [Analyze queries](../../../explore/query-1-performance/explain-analyze)
```
Comment (ddhodge) [inline docs/content/preview/develop/learn/data-modeling-ysql.md:84]:
```suggestion
To explain the behavior of the queries, the examples use **explain (analyze, dist, costs off)**. In practice, you do not need to do this unless you are trying to optimize performance. For more details, see [Analyze queries](../../../explore/query-1-performance/explain-analyze).
```

**Review by aishwarya24 — COMMENTED** [review]
For the following files, change the data modeling link :
- live-fall-back.md -  Line 70 Review [data modeling](../../reference/data-modeling/) strategies.

**Comment by premkumr** [comment]
fixed the links @aishwarya24 .

**Pull request merged**